interface/src/app/analyze/analysis/analysis.js

```diff
@@ -202,7 +202,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
         'properties': {
           'enter': {
             // TODO dynamic layout: compute these constants also (see above)
-            'text': {'value': 'node'},
+            'text': {'value': 'signature'},
```

Comment:
Hmm. This one might be a bug that's just now showing up because the way your database was loaded is different from mine. If my hunch is correct, you would never have been able to run this clustering the data from your database before.

I'm digging deeper, but might not resolve it before I have to do school pick-up. Will have to clone your branch and try to replicate.